multidim-interop/dockerBuildWrapper.sh

```diff
@@ -1,13 +1,45 @@
 #!/usr/bin/env /bin/bash
+set -eou pipefail
+set -x
 
+# Print the hash of our current working directory.
+# We'll use this to store the build cache in S3.
+CWD_HASH=$(tar --sort=name \
+      --mtime="1970-01-01 00:00Z" \
+      --owner=0 --group=0 \
+      -cvf - . | sha256sum | sed 's/.$//' | xargs) # Sed removes the trailing dash, xargs removes the spaces
```

Comment:
This hashes the contents of all files, correct?

Reply:
Yes. And the structure